src/components/App.js

```diff
@@ -263,7 +263,7 @@ const App = () => {
                   <Link to="/audit">Audit</Link>
                 </Menu.Item>
                 {console.log(isCountryBanned)}
```

Comment:
`{console.log(isCountryBanned)}`

Useful for debugging but should be removed for prod. Is logged constantly and with no context.